conformance/tests/specialtypes_any.py

```diff
@@ -87,4 +88,99 @@ def method1(self) -> int:
 assert_type(a.method2(), Any)
 assert_type(ClassA.method3(), Any)
 
+# > When ``Any`` is present in the bases of a type,
+# > it should be considered only after all other known types in the MRO.
 
+class ClassKnown:
+
+    classvar1 = ""
+
+    def __iter__(self) -> Iterator[str]:
+        yield from self.attr1
+
+    def __init__(self):
+        self.attr1: str = ""
+
+    def method1(self) -> str:
+        return ""
+
+class AnyFirst(Any, ClassKnown):
+
+    def method2(self) -> str:
+        return ""
+
+class AnyLast(ClassKnown, Any):
+    def method2(self) -> str:
+        return ""
+
+class GetattrKnown(ClassKnown):
+    def __getattr__(self, name: str) -> int:
+        return 1
+
+class AnyFirstGetAttr(Any, GetattrKnown):
+    def method2(self) -> str:
+        return ""
+
+class AnyLastGetAttr(GetattrKnown, Any):
+    def method2(self) -> str:
+        return ""
+
+class AnySub(Any):
+    ...
+
+# primarily included to demonstrate intent that this is for the full MRO
+class AnySubFirst(AnySub, ClassKnown):
+    def method2(self):
+        ...
+
+
+
+af = AnyFirst()
+assert_type(af.method1(), str)
+assert_type(af.method2(), str)
+assert_type(af.attr1, str)
+assert_type(af.non_exist_method(), Any)
+assert_type(af.non_exist_attr, Any)
+assert_type(af.classvar1, str)
+assert_type(AnyFirst.classvar1, str)
+assert_type(iter(af()), Iterator[str])
+
+al = AnyLast()
+assert_type(al.method1(), str)
+assert_type(al.method2(), str)
+assert_type(al.attr1, str)
+assert_type(al.non_exist_method(), Any)
+assert_type(al.non_exist_attr, Any)
+assert_type(al.classvar1, str)
+assert_type(AnyLast.classvar1, str)
+assert_type(iter(al()), Iterator[str])
+
+af_getattr = AnyFirstGetAttr()
+
+assert_type(af_getattr.method1(), str)
+assert_type(af_getattr.method2(), str)
+assert_type(af_getattr.attr1, str)
+assert_type(af_getattr.triggers_getattr, int)
+assert_type(af_getattr.classvar1, str)
+assert_type(AnyFirstGetAttr.classvar1, str)
+assert_type(iter(af_getattr()), Iterator[str])
+
+al_getattr = AnyLastGetAttr()
+
+assert_type(al_getattr.method1(), str)
+assert_type(al_getattr.method2(), str)
+assert_type(al_getattr.attr1, str)
+assert_type(al_getattr.triggers_getattr, int)
+assert_type(al_getattr.classvar1, str)
+assert_type(AnySubFirst.classvar1, str)
```

Comment:
seems like this is swapped with below? no functional issue, but odd from a readability perspective
```suggestion
assert_type(AnyLastGetAttr.classvar1, str)
```